apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -235,6 +235,9 @@ type VSHNPostgreSQLStatus struct {
 	NetworkPolicyConditions      []v1.Condition `json:"networkPolicyConditions,omitempty"`
 	LocalCAConditions            []v1.Condition `json:"localCAConditions,omitempty"`
 	CertificateConditions        []v1.Condition `json:"certificateConditions,omitempty"`
+	// ObservedGeneration represents the .metadata.generation that the condition was set based upon.
+	// For instance, if .metadata.generation is currently 12, but the .status.conditions[x].observedGeneration is 9, the condition is out of date with respect to the current state of the instance.
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
```

Comment:
Is this automatically managed by K8s?

Reply:
It doesn't look like we need to add this manually. The Crossplane code generator added it in the CRD yamls.

Reply:
fixed it